Dockerfile-core

```diff
@@ -82,16 +79,17 @@ RUN cd /usr/lib/node_modules/npm; npm install fs-extra;        \
 USER docker
 
 
-RUN npm install hubot@2.x;                \
-    npm install hubot-test-helper@^1.3.0; \
-    npm install amqp@^0.2.6;              \
-    npm install d3@^4.2.7;                \
-    npm install jsdom@^9.8.0;             \
-    npm install svg2png@^4.0.0;           \
-    npm install fuzzyset.js@0.0.1;        \
-    npm install nlp_compromise@^6.5.0;    \
-    npm install request@^2.75.0;          \
-    npm install request-promise@^4.1.1;
+RUN npm install coffee-script@^1.12.6    \
```

Comment:
I was trying to remove some warnings, but the warning remained.  After your comment, I have removed this... How does this PR look now?